### PR TITLE
[INLONG-5495][SDK] Support multi-topic manager 

### LINKED ISSUE #5495: [Feature][SDK] Support multi-topic manager
### Description

Current consume policy is one kafka/pulsar consumer only consume one topic. 
Multi-topic policy is applied to those cases that all topics under this task share the same consumer strategy (such as roll back)

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #5495

### Motivation

Current consume policy is one kafka/pulsar consumer only consume one topic.
Multi-topic policy is applied to those cases that all topics under this task share the same consumer strategy (such as roll back)

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Comment by vernedeng** [comment]
this pr is based on #5800 , please review it first

File: inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/manager/InlongMultiTopicManager.java [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+
+    private class UpdateMetaDataThread extends PeriodicTask {
+
+        public UpdateMetaDataThread(long runInterval, TimeUnit timeUnit) {
+            super(runInterval, timeUnit, context.getConfig());
+        }
+
+        @Override
+        protected void doWork() {
+            logger.debug("InLongTopicManagerImpl doWork");
```
Comment (luchunliang) [inline inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/manager/InlongMultiTopicManager.java:265]: logger need to upper case.

**Review by luchunliang — APPROVED** [review]
LGTM

File: inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/manager/InlongMultiTopicManager.java (hunk 1 above)
Comment (vernedeng) [reply inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/manager/InlongMultiTopicManager.java:265]: The logger is inherited from base class ***PeriodicTask***, and it is not a static variables

**Review by doleyzi — APPROVED** [review]

File: inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/manager/InlongMultiTopicManager.java [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+                    try {
+                        succ = topicFetcher.close();
+                    } catch (Exception e) {
+                        LOGGER.error(e.getMessage(), e);
+                    }
+                }
+                LOGGER.info("close fetcher{} {}", fetchKey, succ);
+            }
+        } catch (Exception e) {
+            LOGGER.error(e.getMessage(), e);
```
Comment (healchow) [inline inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/manager/InlongMultiTopicManager.java:138]: Suggest adding more detailed info in the error log.

Comment (vernedeng) [reply inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/manager/InlongMultiTopicManager.java]: fixed, thx

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**